radio/analyzer_engine/commands.py

```diff
@@ -959,6 +959,26 @@ def handle_print(self, message):
         if not isinstance(message, basestring):
             raise Exception('Print command requires string argument. Got: "{}"'.format(type(message)))
         return NoTelecommand('Print', Duration(0), 'Print: "{}"'.format(message))
+    
+    def handle_send_loop(self, loop_arguments):
+        REQUIRED_NO_ARGUMENTS = 2
+
+        try:
+            telecommand = loop_arguments[0]
+
+            if not len(loop_arguments) == REQUIRED_NO_ARGUMENTS:
+                raise Exception('SendLoop command requires two arguments in a list [telecommand, iteration_seconds]. Got: {} arguments.'.format(len(loop_arguments)))
+        
+            iteration_seconds = loop_arguments[1]
+        except TypeError:
+            raise Exception('SendLoop command requires two arguments in a list [telecommand, iteration_seconds]. It is not a list.')
+        
+        if not isinstance(iteration_seconds, int):
+            raise Exception('SendLoop command requires two arguments in a list [telecommand, iteration_seconds]. Got wrong iteration_seconds: "{}"'.format(type(iteration_seconds)))
+
+        telecommand_data = TelecommandData(telecommand)
+
+        return NoTelecommand(telecommand_data.telecommand_name() + ' in SendLoop', Duration(0), 'SendLoop repeats a telecommand in every {} seconds'.format(iteration_seconds))
```

Comment:
Right now SendLoop is not accounted in the analyzer statistics. NoTelecommand should not be returned here. I'd say per minimum this function should return the same object that is returned by the get_telecommand_data function

Reply:
You're right :+1:

Reply:
Fixed :white_check_mark: